extensions/panache/hibernate-orm-panache-common/deployment/pom.xml

```diff
@@ -33,6 +33,16 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-hibernate-orm-panache-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-envers-deployment</artifactId>
```

Comment:
It *is* but i'm not sure it's necessary any longer.  some later changes in dependency trees might have resolved the error encountered here.  I will double check because I felt a bit squeamish about it, too.

Reply:
It's there to provide access to `EnhancerConstants.PERSISTENT_FIELD_WRITER_PREFIX` in the class visitor.  If there's a more appropriate quarkus artifact (or otherwise) i'm happy to swap that out.  But it's literally for that one constant which is mildly gross but less so than copying that value in directly, I think.

Reply:
IIRC that comes from hibernate orm, not envers.

Reply:
ok.  i'll play with that.  thanks.

Reply:
that's exactly what I needed.  thanks.